web-server/src/hooks/useTableSort.ts

```diff
@@ -101,10 +101,44 @@ export const useTableSort = <T = Record<string, any>>(
     return result;
   }, [conf.order, list, simpleSort]);
 
+  const handleCommitSort = useCallback(() => {
+    const result = [...list] as any[];
+    if (!result.some((pr) => pr?.commits)) return simpleSort();
+    if (conf.order === 'asc') {
+      result.sort((a, b) => {
+        return Number(a.commits) - Number(b.commits);
+      });
+    } else {
+      result.sort((b, a) => {
+        return Number(a.commits) - Number(b.commits);
+      });
+    }
+    return result;
+  }, [conf.order, list, simpleSort]);
```

Comment:
Turns out this is not needed, commit sort works fine as is.

Reply:
Okay! I will make the necessary changes.

Reply:
Great! Make sure you update the PR description and title (might need to remove commit based snapshots)

Reply:
I have updated the PR description and title, kindly check the latest commit.

Reply:
Great stuff, merging this!